examples/form/src/App.js

```diff
@@ -62,8 +62,8 @@ class App extends Component {
   }
 
   onChangeMultiplePicker(label, value) {
-    let actives = this.state.multiple_picker;
-    if(actives.indexOf(value) !== -1) {
+    const actives = this.state.multiple_picker;
```

Comment:
I thought that when you define a const is because you don't want change the value, but if is an array or object we can use modify with array and object methods. Do you think that is good idea use the airbnb convention?

Reply:
@iburgos72 https://mathiasbynens.be/notes/es6-const

I think this is kind of confusing but its ok